src/main/java/com/gmail/yuramitryahin/controller/AuthenticationController.java

```diff
@@ -0,0 +1,22 @@
+package com.gmail.yuramitryahin.controller;
+
+import com.gmail.yuramitryahin.model.dto.UserRequestDto;
+import com.gmail.yuramitryahin.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(UserRequestDto userRequestDto) {
```

Comment:
how do we get this dto?

Reply:
what do you mean ?

Reply:
how do you pass this dto to the method? have you tested it?

Reply:
dto comes as json and parsed

Reply:
all working, I tested

Reply:
in dto I have two fields email and password

Reply:
FIXED